### ISSUE TITLE
ykfde-suspend issues  on resume

### ISSUE THREAD

**Issue by bagnaram** [agherzan/yubikey-full-disk-encryption#79]
After enabling ykfde-suspend I seem to experience  some RAM issues  with applications that are open. These problems more often than not occur after longer suspends. (> 5 minutes). For instance my desktop env (sway) will sometimes not draw context menus and some UI applications will seemingly crash until I force reboot.

**Comment by Vincent43** [comment]
Yes, ykfde-suspend is an experimental feature and such issues can happen. For now the only workaround is to disable it.

**Comment by agherzan** [comment]
Related to https://github.com/agherzan/yubikey-full-disk-encryption/issues/77
